asapdiscovery-docking/asapdiscovery/docking/scripts/run_docking_oe.py

```diff
@@ -409,18 +417,29 @@ def get_args():
         action="store_true",
         help="Whether to use GAT model to score docked poses.",
     )
+    parser.add_argument(
+        "-log",
+        "--log_name",
+        type=str,
+        default="run_docking_oe",
+        help="Base name for high-level log file. Defaults to run_docking_oe, "
+        "which enables propagation of log messages to the root logger.",
+    )
 
     return parser.parse_args()
 
 
 def main():
     args = get_args()
+    global log_name
```

Comment:
Why do we need this `global` ? I might be missing something but do we access it from global scope ever? Also seems to be defined in two places.  If possible better to define once (non-global) in `main` pass as an argument to whatever you need

Reply:
@hmacdope as discussed i'm enabling this now

Reply:
done here: https://github.com/choderalab/asapdiscovery/pull/213/commits/021ee5a4692764931e4ccbfe476390e98dd15508